src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,10 @@
+package wolox.training.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import wolox.training.models.Book;
+
+@Repository
+public interface BookRepository extends JpaRepository<Book, Long> {
+    Book findBookByAuthor(String author);
```

Comment:
The Book should be wrapped inside an Optional (Optional<Book>) that way in the future you'll avoid null checks 😄

Reply:
thanks, i though the same but later when created CRUD methods :sweat_smile: , in the next PR i changed it